libr/lang/p/pipe.c

```diff
@@ -231,15 +235,29 @@ static int lang_pipe_run(RLang *lang, const char *code, int len) {
 	hproc = myCreateChildProcess (code);
 	bool connected = false;
 	if (hproc) {
+		hConnected = CreateEvent (NULL, FALSE, FALSE, NULL);
```

Comment:
initialization of all the synchronization stuff should be done before calling `myCreateChildProcess`

Reply:
ok e68a0bc3fb762c4d4090cbd85d7d4fa13ea383c0